### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by whysoserious**
I just started learning ZIO and this g8 template helped me a lot! 
It introduces a couple of small fixes - see commit log.

### CODE REVIEW

File: src/main/g8/src/main/scala/$package$/Boot.scala [hunk 1]
```diff
@@ -55,15 +55,28 @@ object Boot extends App {
 
   def run(args: List[String]): ZIO[zio.ZEnv, Nothing, Int] =
     loadConfig.flatMap { rawConfig =>
-      val configLayer = TypesafeConfig.fromHocon(rawConfig, AppConfig.description)
+      val configLayer = ZLayer.fromEffect {
+        ZIO.fromEither {
+          TypeSafeConfigSource
```
Comment (vpavkin) [inline src/main/g8/src/main/scala/$package$/Boot.scala:60]: Should look much simpler if we use `TypesafeConfig.fromTypesafeConfig` like this:
```scala
TypesafeConfig.fromTypesafeConfig(rawConfig, AppConfig.descriptor)
```

Or am I missing something here?

Comment (whysoserious) [reply src/main/g8/src/main/scala/$package$/Boot.scala:60]: This function no longer exists in `zio-config` `1.0.0-RC16-1`

Comment (vpavkin) [reply src/main/g8/src/main/scala/$package$/Boot.scala:60]: I'm pretty sure it is :thinking: 

https://github.com/zio/zio-config/blob/master/typesafe/src/main/scala/zio/config/typesafe/TypesafeConfig.scala#L27

Comment (whysoserious) [reply src/main/g8/src/main/scala/$package$/Boot.scala:60]: Ah sorry, I just found it. Let me fix that.

**Comment by jczuchnowski** [comment]
@whysoserious please run `scalafmt`

File: src/main/g8/src/main/scala/$package$/Boot.scala (hunk 1 above)
Comment (whysoserious) [reply src/main/g8/src/main/scala/$package$/Boot.scala]: @vpavkin fixed. I didn't find this object due to the typo:
`TypesafeConfig` vs `TypeSafeConfigSource` :)

**Comment by jczuchnowski** [comment]
> @whysoserious please run `scalafmt`

ok, nvm, scalafmt doesn't work on the template level (yet)

File: src/main/g8/src/main/scala/$package$/Boot.scala (hunk 1 above)
Comment (whysoserious) [reply src/main/g8/src/main/scala/$package$/Boot.scala]: I'll let `zio-config` devs know about that.

**Pull request merged**